Source/Fuse.Controls.ScrollView/Scroller.uno

```diff
@@ -198,9 +200,26 @@ namespace Fuse.Gestures
 		{
 			get { return _scrollable == null ? GesturePriority.Lower : _scrollable.GesturePriority; }
 		}
+
+		bool _pressed;
+		void StartInvalidateVisual()
+		{
+			if (!_pressed)
+				UpdateManager.AddAction(_scrollable.InvalidateVisual);
```

Comment:
@mortoray: I actually don't think this would hurt performance. We'll still be able to cache the contents of the scroll-viewer, which is a much better candidate IMO.

Maybe another alternative would be to make CachingMode=Never the default for ScrollView?